docs/CHANGELOG.rst

```diff
@@ -12,6 +12,18 @@ Unreleased
 
 Changed
 -------
+- **BACKWARD INCOMPATIBLE:** Remove obsolete processes: ``bsmap``,
```

Comment:
Great cleaning!! Did you remove ``wgbs.yml`` workflow? It should be also removed and stated in changelog.